Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -504,3 +504,26 @@ extension SPAppDelegate {
         WidgetController.syncWidgetDefaults(authenticated: authenticated, sortMode: sortMode)
     }
 }
+
+// MARK: - Sustainer migration
+extension SPAppDelegate {
+    @objc
+    func migrateSimperiumPreferencesIfNeeded() {
+        guard UserDefaults.standard.bool(forKey: .hasMigratedSustainerPreferences) == false else {
+            return
+        }
+
+        guard isFirstLaunch() == false else {
+            UserDefaults.standard.set(true, forKey: .hasMigratedSustainerPreferences)
+            return
+        }
+
+        NSLog("Migrating Simperium Preferences object to include was_sustainer value")
+        UserDefaults.standard.removeObject(forKey: Simperium.preferencesLastChangedSignatureKey)
+        let prefs = simperium.preferencesObject()
+        prefs.ghostData = ""
+        simperium.saveWithoutSyncing()
+
+        UserDefaults.standard.set(true, forKey: .hasMigratedSustainerPreferences)
```

Comment:
You can prob. move this into a `defer` clause, and remove duplication between L517 and L527